Outrider/1.20.0/Dockerfile

```diff
@@ -0,0 +1,19 @@
+################## BASE IMAGE ######################
+FROM --platform=linux/amd64 quay.io/biocontainers/bioconductor-outrider:1.18.1--r43hf17093f_0
```

Comment:
The base image is using version `1.18.1--r43hf17093f_0`. Folder name `Outrider/1.20.0/` doesnot match.